docs/multi-echo.rst

```diff
@@ -125,50 +126,53 @@ isn't a non-trivial loss in SNR or an increase of artifacts.
 
 Weighted averaging may lead to an increase in SNR
 `````````````````````````````````````````````````
-Multiple studies have shown that a
-weighted average of the echoes to optimize T2* weighting, sometimes called "optimally combined,"
-gives a reliable, modest boost in data quality.
-The optimal combination of echoes can currently be calculated in several software packages including AFNI,
-fMRIPrep, and tedana. In tedana, the weighted
-average can be calculated with `t2smap`_ If no other
-acquisition compromises are necessary to acquire multi-echo data, this boost is worthwhile.
+Multiple studies have shown that a weighted average of the echoes to optimize
+T2* weighting, sometimes called "optimally combined," gives a reliable, modest
+boost in data quality.
+The optimal combination of echoes can currently be calculated in several
+software packages including AFNI, fMRIPrep, and tedana.
+In tedana, the weighted average can be calculated with
+:py:func:`tedana.workflows.t2smap_workflow`.
+If no other acquisition compromises are necessary to acquire multi-echo data,
+this boost is worthwhile.
 
 Consider the life of the dataset
 ````````````````````````````````
 If other compromises are necessary, consider the life of the data set.
-If data is being acquired for a discrete
-study that will be acquired, analyzed, and published in a year or two, it might not be worth making
-compromises to acquire multi-echo data.
-If a data set is expected to be used for future analyses in later
-years, it is likely that more powerful approaches to multi-echo denoising will sufficiently mature and add
-even more value to a data set.
-
-Other multi-echo denoising methods, such as MEICA, the predecessor to tedana, have shown the potential for
-much greater data quality improvements, as well as the ability to more accurately separate visually similar
-signal vs noise, such as scanner based drifts vs slow changes in BOLD signal.
-More powerful methods are
-still being improved, and associated algorithms are still being actively developed.
-Users need to have the time and knowledge to look
-at the denoising output from every run to make sure denoising worked as intended.
+If data is being acquired for a discrete study that will be acquired, analyzed,
+and published in a year or two, it might not be worth making compromises to
+acquire multi-echo data.
+If a data set is expected to be used for future analyses in later years, it is
+likely that more powerful approaches to multi-echo denoising will sufficiently
+mature and add even more value to a data set.
+
+Other multi-echo denoising methods, such as MEICA, the predecessor to tedana,
+have shown the potential for much greater data quality improvements, as well as
+the ability to more accurately separate visually similar signal vs noise, such
+as scanner based drifts vs slow changes in BOLD signal.
+More powerful methods are still being improved, and associated algorithms are
```

Comment:
This was also my language, but it's too wordy. Suggest changing too:
"More powerful methods, and associated algorithms are..." (cutting "are still being improved")

Reply:
```suggestion
More powerful methods and associated algorithms are
```